godot/CellularAutomata/03.DungeonGenerator/ButtonCameraSwitch.gd

```diff
@@ -1,14 +1,14 @@
 extends Button
 
-export (NodePath) var camera_one_path
-export (NodePath) var camera_two_path
+@export var camera_one_path : NodePath
```

Comment:
`@export` now supports the `Node` type directly so there's no need for `NodePath` any longer. These exports and `@onready` will become:

```py
@export var camera_one: Camera2D
@export var camera_two: Camera2D
```

We don't use underscore (`_`) with exports because they are treated like public variables that we have access to (from the Inspector at the very least in this case). So the rest of the code needs to be updated to reflect this.

Using node types instead of `NodePath` has the added benefit that we can now specify a given node type like `Camera2D` and the system grays out the rest in the node selection pop-up:

![image](https://user-images.githubusercontent.com/1177508/225689265-2ef0655d-470a-47a7-a574-50734dab854a.png)